AzureCalling/Controllers/CallViewController.swift

```diff
@@ -521,10 +522,46 @@ class CallViewController: UIViewController, UICollectionViewDelegate, UICollecti
     }
 
     @objc func recordingActiveChangeUpdated(_ notification: Notification) {
-        guard let isRecordingActive = callingContext?.isRecordingActive else {
+        guard let isRecordingActive = callingContext?.isRecordingActive,
+              let isTranscriptionActive = callingContext?.isTranscriptionActive else {
             return
         }
-        let notificationText = isRecordingActive ? meetingRecordingActiveText : meetingRecordingStopText
+
+        let notificationText: NSAttributedString = {
+            switch (isRecordingActive, isTranscriptionActive) {
+            case (true, false):
+                return meetingRecordingActiveText
+            case (true, true):
+                return meetingRecordingAndTranscriptionActiveText
+            case (false, true):
+                return meetingRecordingStopTranscriptionActiveText
+            default:
+                return meetingRecordingStopText
+            }
+        }()
+
+        messageBannerView.showBannerMessage(notificationText)
```

Comment:
We can create a private func and reuse this logic for both.

Reply:
Not sure if one logic function can be applied to the events.
Since the text is different based on triggering event. For example, if isRecordingActive = true, isTranscriptionActive = false. If it is in the Recording listener, we would display "Recording start", if it's trigger from Transcription change, it would display "Transcription stops but recording is continuing"

Reply:
We always update the message based on the values of both `isRecordingActive` and `isTranscriptionActive`, and at this point those values are already updated in `CallingContext`. So we should be able to reuse the same logic.

Reply:
Updated with a helper function. Also changed the base branch to target `feature/teams_interop`